triton/scripts/build_librispeech_pruned_transducer_stateless3_offline.sh

```diff
@@ -58,15 +58,11 @@ if [ ${stage} -le -1 ] && [ ${stop_stage} -ge -1 ]; then
     echo "export onnx"
     cp scripts/*onnx*.py ${recipe_dir}/
```

Comment:
We may not need this cp anymore, if we are trying use the icefall native export-onnx.py.

Reply:
Yeah sure!